### PR TITLE
Fixes tag-and-push command

### PR DESCRIPTION
**Pull request by arvy**

### CODE REVIEW

File: publish.sh [hunk 1]
```diff
@@ -18,7 +18,9 @@ do
   if [ "$1" = "local" ]; then
     docker tag bigquery-fixtures:${project_id}_${HASH} bigquery-fixtures:${project_id_underscores}_latest
   else
-    ./teads-central docker tag-and-push --image bigquery-fixtures:${project_id_underscores}_${HASH} --custom-tags ${project_id_underscores}
+    ./teads-central docker tag-and-push	--image bigquery-fixtures \
+			--custom-tags ${project_id_underscores}_${HASH} \
```
Comment (gbougeard) [inline publish.sh]: Will you really use this `${project_id_underscores}_${HASH}` tag? or maybe a timestamped tag would be more readable/useful?

Comment (arvy) [inline publish.sh]: ..you mean instead of the HASH?

Comment (gbougeard) [inline publish.sh]: yes, if the goal of this tag is to track history, maybe a timestamp might be more useful than a hash

Comment (arvy) [inline publish.sh]: I think the goal is to be able to identify which image we're dealing with (when troubleshooting an issue).

Comment (arvy) [inline publish.sh]: The `${project_id_underscores}` tag will serve as the `master`/`latest` tag

**Pull request merged**